common/persistence/nosql/nosql_execution_store.go

```diff
@@ -598,8 +598,13 @@ func (d *nosqlExecutionStore) GetTransferTasks(
 		return nil, convertCommonErrors(d.db, "GetTransferTasks", err)
 	}
 
+	var tTasks []*persistence.TransferTaskInfo
```

Comment:
nit: Since the number of elements are known you can initialize this slice like 
`tTasks := make(*persistence.TransferTaskInfo, len(tasks)`

same in a few other places